test/any/client/api_matrix-test.es6.js

```diff
@@ -18,41 +18,71 @@
 
 describe('ApiMatrix', function() {
   let apiMatrix;
+  let browser = function(browserName, browserVersion,
+    osName, osVersion) {
+      return org.chromium.apis.web.Browser.create({
+        browserName,
+        browserVersion,
+        osName,
+        osVersion,
+      });
+    };
+  let webInterface = function(interfaceName, apiName) {
+    return org.chromium.apis.web.WebInterface.create({
+      interfaceName,
+      apiName,
+    });
+  };
   let browserAPI = function(browserName, browserVersion,
     osName, osVersion, interfaceName, apiName) {
-      return org.chromium.apis.web.BrowserAPI.create({
-        browserName: browserName,
-        browserVersion: browserVersion,
-        osName: osName,
-        osVersion: osVersion,
-        interfaceName: interfaceName,
-        apiName: apiName,
+      return org.chromium.apis.web.BrowserWebInterfaceJunction.create({
+        sourceId: `${browserName}_${browserVersion}_${osName}_${osVersion}`,
+        targetId: `${interfaceName}#${apiName}`,
       });
     };
   beforeEach(function() {
-    let browserApiDao = foam.dao.EasyDAO.create({
+    let browserApiDAO = foam.dao.EasyDAO.create({
```

Comment:
I found the browser.interface.put() can not find the correct junctionDAO if the junctionDAO is not declared as a property inside a foam class.